src/toil/wdl/wdltoil.py

```diff
@@ -708,6 +705,94 @@ def is_url(filename: str, schemes: List[str] = ['http:', 'https:', 's3:', 'gs:',
                 return True
         return False
 
+def import_url_files(environment: WDLBindings, file_source: Union[AbstractFileStore, Toil], path: Optional[List[str]] = None, include_self: bool = False) -> Optional[Dict[str, str]]:
+    """
+    Iterate through the environment and import all files from the possible URLs. This will not import if the file is detected as local.
```

Comment:
I think maybe the concept we want here is "remote files" vs. "local files" and not "URL files" vs. "local files". A `file://` URI is a URI, and unless we think it's not *actually* a "URL" (due to not really locating it on the Internet) and we want to rely on people knowing that, I don't think we want to talk about "URL files" as a category that doesn't include `file://`.

We also might end up having to deal with something like `http://localhost:8000/file.txt`, which is a URL but which is also *really* a local thing we're going to want to be fetching only on the leader.